src/main/webapp/app/course/tutorial-groups/tutorial-groups-management/tutorial-groups/tutorial-groups-management/tutorial-groups-import-dialog/tutorial-groups-registration-import-dialog.component.ts

```diff
@@ -232,6 +243,11 @@ export class TutorialGroupsRegistrationImportDialogComponent implements OnInit,
                     firstName: csvRow[usedFirstNameHeader]?.trim() || '',
                     lastName: csvRow[usedLastNameHeader]?.trim() || '',
                 } as StudentDTO;
+                registration.campus = csvRow[usedCampusHeader]?.trim() || '';
+                registration.capacity = csvRow[usedCapacityHeader] ? parseInt(csvRow[usedCapacityHeader], 10) : undefined;
```

Comment:
Please use the Number constructor instead, or is there a particular reason why we cannot use it?

```suggestion
                registration.capacity = csvRow[usedCapacityHeader] ? Number(csvRow[usedCapacityHeader]) : undefined;
```

Reply:
done 👍